src/mine_loop.rs

```diff
@@ -656,8 +656,7 @@ pub(crate) async fn create_block_transaction_from(
             vm_job_queue.clone(),
             job_options.clone(),
         )
-        .await
-        .expect("Must be able to merge transactions in mining context");
+        .await?; // fix #579.  propagate error up.
```

Comment:
I don't think issue tracker information of fixed bugs should be in the code.

Reply:
We appear to have a difference of opinion there.  It is something I have always done, for decades, because the issue in the tracker inevitably holds more context and information than can go into a code-comment.   Thus a reference serves as a pointer to that additional context and can be crucial for later readers of the code to understand the history and reasoning for a change.

Reply:
keeping this comment.

Reply:
Fair points. :+1: